ConsumerApi.Tests.Integration/Support/Constants.cs

```diff
@@ -2,5 +2,13 @@
 
 public static class Constants
 {
+    public const string CLIENT_ID = "test";
+    public const string CLIENT_SECRET = "test";
+
+    public const string VALID_CHALLENGE_ID = "CHLjVPS6h1082AuBVBaR";
```

Comment:
Can be removed once you fixed one of my previous comments

Reply:
Changes made.